src/WebApi/Application/Repositories/IInterviewQuestionRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Domain.Entities;
+using Domain.Models;
+
+namespace Application.Repositories
```

Comment:
Let's convert all namespaces to the file scoped version. We will enforce it as a standard across whole repository. 

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces